packages/async/src/babel-plugin.ts

```diff
@@ -0,0 +1,143 @@
+import * as Types from '@babel/types';
+import {NodePath, Binding} from '@babel/traverse';
+
+const DEFAULT_PROCESS_PACKAGES = {
```

Comment:
are these packages to process or packages *that* process? Maybe call them `processorPackageNames`?

Reply:
They are packages that get processed, will rename for clarity